base/reduce.jl

```diff
@@ -637,42 +637,37 @@ function contains(eq::Function, itr, x)
     return false
 end
 
-
-## countnz & count
-
 """
     count(p, itr) -> Integer
+    count(itr)
```

Comment:
I don't think most of the Base docstrings have made the `::` switch yet. IMO better to wait and do them all at once, keeping `->` in the interim.